server/opts.go

```diff
@@ -2310,6 +2311,8 @@ func parseJetStream(v any, opts *Options, errors *[]error, warnings *[]error) er
 					return &configErr{tk, fmt.Sprintf("%s %s", strings.ToLower(mk), err)}
 				}
 				opts.JetStreamMaxCatchup = s
+			case "account_nrg":
```

Comment:
ngr means something to us, but maybe not to users, can we make something a bit more descriptive here?

Reply:
I'm not at all attached to the name, suggestions welcome! 😄